Assets/Extensions/Unity3D-Raven-CS/Unity3DRavenCS.cs

```diff
@@ -256,13 +257,19 @@ public IEnumerator AsyncSend()
             }
 
             startTime = Time.time;
-            www = new WWW(m_dsn.sentryUri, requestBuffer, headers);
-            yield return www;
+            uwr = new UnityWebRequest(m_dsn.sentryUri);
```

Comment:
`
uwr = new UnityWebRequest(m_dsn.sentryUri, UnityWebRequest.kHttpVerbPOST);
`

It has to be a POST request.

Reply:
Ah I see, let me change that.

Reply:
@defmys latest commit should have that. I think the headers are already being set so the rest of it should be okay.